kegtab/src/main/res/xml/settings_general.xml

```diff
@@ -41,6 +41,27 @@
             android:summaryOn="Temperatures will be shown in Celsius"
             android:title="Show Temperatures in Celsius">
         </CheckBoxPreference>
+        <CheckBoxPreference
+            android:title="Show ABV when zero"
+            android:key="config:ABV_DISPLAY_WHEN_ZERO"
+            android:summaryOn="ABV will be displayed even when zero"
+            android:summaryOff="ABV will not be displayed when zero"
+            android:defaultValue="false">
+        </CheckBoxPreference>
+        <CheckBoxPreference
+            android:title="Show IBU when zero"
+            android:key="config:IBU_DISPLAY_WHEN_ZERO"
+            android:summaryOn="IBU will be displayed even when zero"
+            android:summaryOff="IBU will not be displayed when zero"
+            android:defaultValue="false">
+        </CheckBoxPreference>
+        <CheckBoxPreference
+            android:title="Show Tap Notes"
+            android:key="config:DISPLAY_TAP_NOTES"
+            android:summaryOn="Tap Notes will be displayed if available"
+            android:summaryOff="Tap Notes will not be displayed"
+            android:defaultValue="true">
+        </CheckBoxPreference>
```

Comment:
This should probably be in the `User Interface` section of [settings_kegerator.xml](/Kegbot/kegbot-android/blob/master/kegtab/src/main/res/xml/settings_kegerator.xml)

Reply:
Hmm, yeah I could see it making more sense on that page since that grouping (currently only one item) controls the tap detail display. Makes sense under the title of "Look and Feel", but maybe that should just be "Units" and the UI settings are under Kegerator > User Interface